### DISCUSSION TITLE
Bosch Smart Home

### DISCUSSION THREAD

**Discussion by Asky82** [indomus/forum#2876] [Hardware & Firmware]
Ciao a tutti,

su InDomus ho notato che non ci sono recensioni riguardo il sistema/devices Bosch Smart Home. 

Cosa ne pensate dei loro prodotti? 

Un saluto

**Comment by indomus** [comment]
> Ciao a tutti,
> 
> su InDomus ho notato che non ci sono recensioni riguardo il sistema/devices Bosch Smart Home.
> 
> Cosa ne pensate dei loro prodotti?
> 
> Un saluto

Eh, non li abbiamo testati, quindi è difficile avere un'opinione. Il marchio è buono: li testeremo e ci faremo un'idea!
